src/applications/vaos/services/appointment/transformers.js

```diff
@@ -25,9 +25,12 @@ function getAppointmentType(appt) {
     return APPOINTMENT_TYPES.ccRequest;
   } else if (appt.typeOfCareId) {
     return APPOINTMENT_TYPES.request;
-  } else if (appt.clinicId || appt.vvsAppointments) {
+  } else if (
+    appt.vvsAppointments?.length ||
+    (appt.clinicId && appt.communityCare === false)
```

Comment:
We need to change `appt.communityCare === false` to `!appt.communityCare`. Sometimes this value can come back null.

Reply:
will do